### PR TITLE
fix: sphinx version and update footer

### PR DESCRIPTION
**Pull request by Revathyvenugopal162**
- [x] Add the compatibility with sphinx>6
- [x] Add the footer as ansys-sphinx-theme version.
##
image
## Expected output 
image

### CODE REVIEW

**Review by akaszynski — APPROVED** [review]
Saw that `sphinx-design` finally supported Sphinx v6.0

Thanks!

**Review by MaxJPRey — APPROVED** [review]
Nice to see the the footer.

File: src/ansys_sphinx_theme/__init__.py [hunk 1]
```diff
@@ -85,6 +85,29 @@ def setup_default_html_theme_options(app):
     app.config.html_theme_options.setdefault("collapse_navigation", True)
 
 
+def update_footer_theme(app: Sphinx, pagename: str, templatename: str, context, doctree) -> None:
```
Comment (MaxJPRey) [inline src/ansys_sphinx_theme/__init__.py:88]: If possible, can you add the type of the `doctree` parameter.

Comment (RobPasMue) [reply src/ansys_sphinx_theme/__init__.py:88]: And ``context``

**Review by RobPasMue — APPROVED** [review]
LGTM! Thanks @Revathyvenugopal162

File: pyproject.toml [hunk 2]
```diff
@@ -25,15 +25,15 @@ classifiers = [
     "Programming Language :: Python :: 3",
 ]
 dependencies = [
-    "Sphinx>=4.2.0,<6",
+    "Sphinx>=4.2.0",
```
Comment (jorgepiloto) [inline pyproject.toml:28]: Did you check this is not causing any issue in the top navigation bar, @Revathyvenugopal162? I remember we imposed this condition due to this issue.

**Review by jorgepiloto — APPROVED** [review]

File: pyproject.toml (hunk 2 above)
Comment (Revathyvenugopal162) [reply pyproject.toml:28]: Yes, it is fixed, i checked it. https://github.com/pydata/pydata-sphinx-theme/issues/1130 issue in pydata related to that.

**Pull request merged**